README.md

```diff
@@ -19,11 +19,10 @@ since 2011, and has grown to encompass tens of thousands of MySQL nodes.
 
 For more about Vitess, please visit [vitess.io](https://vitess.io).
 
-Vitess has a growing community. You can view the list of adopters
-[here](https://github.com/vitessio/vitess/blob/main/ADOPTERS.md).
+Vitess has a growing community. [View the list of adopters](https://github.com/vitessio/vitess/blob/main/ADOPTERS.md).
 
 ## Reporting a Problem, Issue, or Bug
-To report a problem, the best way to get attention is to create a GitHub [issue](.https://github.com/vitessio/vitess/issues ) using proper severity level based on this [guide](https://github.com/vitessio/vitess/blob/main/SEVERITY.md). 
+To report a problem, create a [GitHub issue](https://github.com/vitessio/vitess/issues) using proper severity level based on [this guide](https://github.com/vitessio/vitess/blob/main/SEVERITY.md). 
```

Comment:
We do not really use severity flag anymore and labeling issues is not possible for non-maintainers. I would be okay to part of the sentence mentioning severity levels.

```suggestion
To report a problem, create a [GitHub issue](https://github.com/vitessio/vitess/issues). 
```